api/sheets/places_in.md

```diff
@@ -1,58 +1,75 @@
 
-## General information about this endpoint
+> **Note**
+>
+> Be sure to follow the instructions for [Installing and Enabling the Sheets Add-On](/api/sheets/) before using this formula.
 
-**Formula**: `=DCPLACESIN(dcids, placeType)`
+## Formula
 
-**Required arguments**:
+```
+=DCPLACESIN(dcids, placeType)
+```
 
-*   `dcids`: A list of (parent) `Place` nodes, identified by their DCIDs.
+## Required Arguments
 
-*   `placeType`: The type of the contained (child) `Place` nodes to filter by. For example, `City` and `County` are contained within `State`. For a
-    full list of available types, see [the Data Commons graph browser entry for `Place`](https://datacommons.org/browser/Place).
+* `dcids`: A list of parent `Place` nodes, identified by their DCIDs.
+* `placeType`: The type of the contained child `Place` nodes to filter by. For example,`City` and `County`are contained within `State`. For a full list of available types, see [the Data Commons graph browser entry for Place](https://datacommons.org/browser/Place).
 
+## Returns
 
-## Assembling the information you will need to use this formula
+Lists of child places from a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html#dcid). Returns a list of child places of the specified place DCIDs, of the specified place type.
 
-This endpoint requires the argument `dcids`. [DCIDs](/glossary.html) are unique node identifiers defined by Data Commons. Your query will need to specify the DCIDs for the parent places of interest.
-
-This endpoint also requires the argument `placeType`, specifying the type of the child places you desire in the response.
+> **Note**:
+> It’s best to minimize the number of function calls to `=DCPLACESIN(dcids, placeType)` by using a single call to get the names for a column of nodes. This is because a spreadsheet will make one call to a Google server [*per custom function call*](https://developers.google.com/apps-script/guides/sheets/functions#optimization). If your sheet contains many separate calls to `=DCPLACESIN(dcids, placeType)` you can expect it to be slow and return with errors.
 
 ## Examples
 
-Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
+This section contains examples of using the `=DCPLACESIN(dcids, placeType)` formula.
+
+### Example 1: Retrieve a List of Counties in Delaware
+
+To retrieve a list of counties in Delaware, perform the following steps:
 
-### Example 1: Retrieve a list of the counties in Delaware.
+1. Place your cursor in the cell where you want to add the DCID for Delaware. In this case, cell A2.
+2. Enter the Delaware DCID of “geoId/10”.
```

Comment:
In line 55, we use bold instead of quotes to show what the user should enter. I personally prefer the bold, but we should probably standardize across examples (and the other pages.)

Reply:
good point. it's been hard to standardize across the PR's, and i can do a sweep after the first versions are in. i'll update these to bold for now though